client/jetpack-cloud/sections/agency-dashboard/dashboard-modal/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { Modal } from '@wordpress/components';
+import classNames from 'classnames';
+import { ReactNode } from 'react';
+
+import './style.scss';
+
+export type DashboardModalProps = {
+	children: ReactNode;
+	className?: string;
+	onClose: () => void;
+	subtitle?: ReactNode;
+	title: string;
+};
+
+export default function DashboardModal( {
```

Comment:
I added a separate Dashboard modal in case we will have another modal without the form logic.